automation/build.sbt

```diff
@@ -19,21 +18,21 @@ cancelable in Global := true
 
 /**
   * sbt forking jvm -- sbt provides 2 testing modes: forked vs not forked.
-  * -- forked: each task (test class) is executed sequentially in a forked JVM.
+  * -- forked: each task (test class) is executed sequentially in a forked jvm.
   *    Test results are segregated, easy to read.
-  * -- not forked: all tasks (test classes) are executed in same sbt JVM.
+  * -- not forked: all tasks (test classes) are executed in same sbt jvm.
   *    Test results are not segregated, hard to read.
   *
   */
 
 /**
-  * Specify that all tests will be executed in a single external JVM.
-  * By default, tests executed in a forked JVM are executed sequentially.
+  * Specify that each test class will be mapped to run in its own jvm.
+  * Default: fork: = false It means all test classes run in one jvm.
```

Comment:
remove this line because it's clear from the above text

Reply:
done